src/vm/dllimport.cpp

```diff
@@ -6118,6 +6104,36 @@ static void DetermineLibNameVariations(const char* const** libNameVariations, in
         *numberOfVariations = COUNTOF(NameOnly);
     }
 }
+#else
```

Comment:
Nit: It would be nice to add `// FEATURE_PAL` comment here.

Reply:
Updated